tensorboard/plugins/core/core_plugin.py

```diff
@@ -90,10 +91,18 @@ def get_plugin_apps(self):
             "/histograms": self._redirect_to_index,
             "/images": self._redirect_to_index,
         }
-        apps.update(self.get_resource_apps())
+        apps.update(self._static_handlers)
+        remappings = {
+            # Serve notifications from static JSON file if present.
+            "/data/notifications": "/notifications_note.json",
```

Comment:
I tried to verify the change and 
`...com/data/notifications_note.json` redirect to `.com/data/notifications_note.json`
which is not the correct path (omit 'data/')

Reply:
I don't quite follow, can you provide a little more detail about the exact way you were testing this and exactly what response you got?  I'm not sure I understand the `...com` versus `.com` part and also I wouldn't expect this remapping to affect an incoming path with `notifications_note.json` (since it only maps the path `/data/notifications`).